tests/browser/heda_settings_system.robot

```diff
@@ -0,0 +1,70 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
+    Select Frame			//iframe[contains(@id, "vfFrameId")]
+    Click Link				//a[contains(text(),'System')]
```

Comment:
This should become a HEDA library keyword like `Select Settings Tab  <label>`

Reply:
This is patterned after Go To Account home.  I removed it and then nothing runs.  I'll look into making this into a HEDA library function in my 2nd round via a WI in order to track this work.